course1/module9/Question2.js

```diff
@@ -0,0 +1,9 @@
+//Go to youtube. Open any video. Add a button to the page using JS. On click of the button, the video playback speed should change to 10x
+
+<button type="button" onclick="abc()">
+  click here
+</button>;
+
```

Comment:
Add the whole logic of adding button to the JS function. So we will have only one JS function which adds the button & on click updates the playback speed. Use add event listener for button